crates/ruff_linter/src/registry/rule_set.rs

```diff
@@ -207,6 +207,15 @@ impl RuleSet {
         *self = set.union(&RuleSet::from_rule(rule));
     }
 
+    #[inline]
+    pub fn set(&mut self, rule: Rule, enabled: bool) {
```

Comment:
This is kind of unrelated but I noticed that we can simplify `configuration.rs` a bit